site/content/how-to/monitoring/dashboard.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "NGINX Plus Dashboard"
+description: "Learn how to view the NGINX Plus dashboard to see real-time metrics."
+weight: 200
+toc: true
+docs: "DOCS-000"
+---
+
+{{<custom-styles>}}
+
+## Overview
+
+The NGINX Plus dashboard offers a real-time live activity monitoring interface that shows key load and performance metrics of your server infrastructure. The dashboard is enabled by default for NGINX Gateway Fabric deployments that use NGINX Plus as the data plane. The dashboard is available on port 8765.
+
+To access the dashboard:
+
+1. Use port-forwarding to forward connections to port 8765 on your local machine to port 8765 on the NGINX Gateway Fabric pod (replace `<nginx-gateway-fabric-pod>` with the actual name of the pod).
+
+    ```shell
+    kubectl port-forward <nginx-gateway-fabric-pod> 8765:8765 -n nginx-gateway
+    ```
+
+1. Open your browser to [http://127.0.0.1:8765/dashboard.html](http://127.0.0.1:8765/dashboard.html) to access the dashboard.
+
+The dashboard will look like this:
+
+{{<img src="img/nginx-plus-dashboard.png" alt="NGINX Plus dashboard">}}
```

Comment:
```suggestion
{{<img src="img/nginx-plus-dashboard.png" alt="">}}
```

If an image does not have descriptive text, the `alt` attribute should be left empty to mark the image as intended for presentation only.

Reply:
Can I omit the alt variable completely? `{{<img src="img/nginx-plus-dashboard.png"}>}`

Reply:
It's permissible but not recommended. In context I would describe it as a decorative image.

https://www.w3.org/WAI/tutorials/images/decorative/

> Decorative images don’t add information to the content of a page. For example, the information provided by the image might already be given using adjacent text, or the image might be included to make the website more visually attractive.
>
> In these cases, a null (empty) alt text should be provided (alt="") so that they can be ignored by assistive technologies, such as screen readers.